module.txt

```diff
@@ -7,6 +7,7 @@
     "serverSideOnly": false,
     "dependencies": [
         { "id": "CoreAdvancedAssets", "minVersion": "1.1.0" },
+        { "id": "Health", "minVersion": "1.0.1", "because": "tests"},
```

Comment:
At least make it optional if it is just for tests...
```suggestion
        { "id": "Health", "minVersion": "1.0.1", "optional": true, "because": "tests"},
```